doc/source/notebooks/08_nodesets_and_node_queries.ipynb

```diff
@@ -0,0 +1,1177 @@
+       "5006      VPL_TC\n",
+       "...          ...\n",
+       "100760    VPL_IN\n",
+       "100761    VPL_IN\n",
+       "100762    VPL_IN\n",
+       "100763    VPL_IN\n",
+       "100764    VPL_IN\n",
+       "\n",
+       "[65198 rows x 1 columns]"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    " circuit.nodes['thalamus_neurons'].get({\"mtype\": {\"$regex\": \"^VPL_.*\"}}, properties=['mtype'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Querying ranges of values\n",
+    "\n",
+    "As shown above, we can query data based on discrete values of properties such as node set, layer etc. However, all properties are not made equal. \n",
+    "\n",
+    "When there is a `list` in the query, it is _usually_ considered an \"OR\" condition. There is one exception and that is numeric properties represented by `float`s. Whenever a property is represented by a `float`, a `list` will need to specify exactly two values: start of a range and end of a range (i.e., minimum and maximum values to consider).\n",
+    "\n",
+    "Let's take a look which properties in `'thalamus_neurons'` are floats:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "@dynamics:holding_current       float32\n",
+       "@dynamics:threshold_current     float32\n",
+       "etype                          category\n",
+       "layer                            object\n",
+       "model_template                   object\n",
+       "model_type                       object\n",
+       "morph_class                    category\n",
+       "morphology                     category\n",
+       "mtype                          category\n",
+       "orientation_w                   float32\n",
+       "orientation_x                   float32\n",
+       "orientation_y                   float32\n",
+       "orientation_z                   float32\n",
+       "region                           object\n",
+       "rotation_angle_xaxis            float32\n",
+       "rotation_angle_yaxis            float32\n",
+       "rotation_angle_zaxis            float32\n",
+       "synapse_class                  category\n",
+       "x                               float32\n",
+       "y                               float32\n",
+       "z                               float32\n",
+       "dtype: object"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "circuit.nodes['thalamus_neurons'].property_dtypes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's take `x`, `y`, `z` and query nodes that inside a 50x50x50 box limited by:\n",
+    "* `100 < x < 150`\n",
+    "* `500 < y < 550`\n",
+    "* `400 < z < 450`"
```

Comment:
Shouldn't be `<=` instead of `<`?

Reply:
Technically, true but less elegant, lol -> changed